app.js

```diff
@@ -363,95 +362,189 @@ d3.select('#clear')
-      var desc = row.insertCell(0);
-      var more = row.insertCell(1);
-     };
-
-     //console.log(data);
-
-     if (data.length == 1) desc.innerHTML =  corridorInfo[item].text + ': ' + '<span class="txt-bold">' + data[0] + '</span> ';
-     else {
-      console.log(data);
-      // TODO: Add additional rows for the sub-items
-      desc.innerHTML =  corridorInfo[item].text + '</br>';
-      for (j in corridorInfo[item].secondaryText) {
-       desc.innerHTML += '<p class=secondaryText>' + corridorInfo[item].secondaryText[j] + ': ' + '<span class="txt-bold">' + data[j] + '%</span></p>';
-       //var secondaryText = document.createElement('p');
-       //secondaryText.innerHTML = '<p>' + corridorInfo[item].secondaryText[j] + ': ' + '<span class="txt-bold">' + data[j] + '</span></p>';
-       //desc.appendChild(secondaryText);
+      var section = d3.select('#corridor-info')
+        .append('div')
+        .attr('id', corridorInfo[item].id)
+
+      var title = section
+        .append('div')
+        .classed('mt12', true);
+
+      title
+        .append('span')
+        .classed('txt-bold', true)
+        .text(corridorInfo[item].text)
+
+      if (data.length === 1){
+        title
+          .append('span')
+          .text(data[0])
+          .attr('class', 'fr')
       }
-     };
+      else {
+
+        var dataRow = section
+          .selectAll('.dataRow')
+          .data(data)
+          .enter()
+          .append('div')
+          .attr('class', 'dataRow py1 mt6')
+          .style('height', '18px')
+
+        dataRow
+          .append('div')
+          .style('width', '30%')
+          .style('display', 'inline-block')
+          .style('float', 'left')
+          .text(function(d,i) {
+            return corridorInfo[item].secondaryText[i]
+          })
+          .attr('class', ' txt-s small py1')
+
+
+        var bar = dataRow
+          .append('div')
+          .style('width', '70%')
+          .style('display', 'inline-block')
+          .style('position', 'relative')
+          .style('float', 'right')
+          .style('height', '100%')
+
+        bar
+          .append('span')
+          .attr('class', 'z5 absolute py1 px6 txt-s')
+          .text(function(d){
+            return d+'%'
+          })
+          .each(function(d,i){
+            var color = i === maxIndex ? 'color-white' : 'color-black'
+            d3.select(this).classed(color, true)
+          })
+
+        bar
+          .append('div')
+          // .attr('class', 'round')
+          .style('width', function(d){
+            return 100 * d/data[maxIndex] + '%'
+          })
+          .style('background-color', '#448ee4')
+          .style('height', '100%')
+          .each(function(d,i){
+            var color = i === maxIndex ? 'bg-blue-light' : 'bg-darken10'
+            d3.select(this).classed(color, true)
+          })
+
+      }
+
+    // if (document.getElementById('corridor-info-table').rows.length > item) {
```

Comment:
Looks like you refactored the side bar and this code is now superfluous? If so, do you want to just remove it now? I'll do a final clean-up of all the code and comments at the end, but it might be helpful to clean this up now.